### ISSUE TITLE
Unable to modify user

### ISSUE THREAD

**Issue by kwschnei** [Peppermint-Lab/peppermint#60]
I attempted to rename the default user by editing it and turning it into my user, but none of the changes would stick. I then made a new user for myself (and misspelled my name in the process). When I attempted to edit my new user, none of the changes I submitted saved for that user either.

I initially suspected that perhaps the logged in user couldn't change their account, but perhaps there's something wrong with the edit functionality entirely.

**Comment by kwschnei** [comment]
Deleting a user also doesn't seem to work as I can't delete my misnamed user nor the default admin user.

**Comment by kwschnei** [comment]
On a related note, using the View Profile page to make changes doesn't seem to work - clicking the Save button doesn't do anything.

**Comment by potts99** [comment]
Looking into this now

**Comment by potts99** [comment]
If you try re pulling the docker image, there should be a better refetching process and show the updates, theyre working for me. If you could let me know how this goes then great :)

**Comment by kwschnei** [comment]
I made the name change via the Users page, and after navigating away and coming back, it shows up there. However, I then went to view the profile (I was logged in as the user I was changing the name of), and it still shows the old name on the Profile page.

Deleting a user still Dows not seem to work. (Also, I wonder if deleting a user is wise - wouldn’t that result in orphaned tickets? Deactivating users/clients might be a better idea.)
